src/components/LoadMoreButton.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { Box, Button } from "@chakra-ui/react";
+import { MotionBig } from "./motion";
+
+interface Props {
+  morePokemon: boolean | undefined;
+  fetchNextPokemonPage: () => void;
+}
+
+function LoadMoreButton({ morePokemon, fetchNextPokemonPage }: Props) {
+  if (morePokemon) {
+    return (
+      <MotionBig>
+        <Box m="2rem">
+          <Button onClick={fetchNextPokemonPage}>Load More</Button>
+        </Box>
+      </MotionBig>
+    );
+  } else {
+    return <></>;
+  }
```

Comment:
We can simply return witout the use of `else` since that will leave the execution scope. https://medium.com/swlh/return-early-pattern-3d18a41bba8